src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -913,4 +913,10 @@ This version used in: {2}</value>
   <data name="Install_version_0" xml:space="preserve">
     <value>Install version '{0}'</value>
   </data>
-</root>
+  <data name="AddMissingStatements" xml:space="preserve">
```

Comment:
That is the one I am using now.  I forgot to remove them from the C# and VB.NET resources from my last PR.